PYME/localization/remFitBuf.py

```diff
@@ -545,6 +558,29 @@ def __call__(self, gui=False, taskQueue=None):
 
     @classmethod
     def calcSigma(cls, md, data):
+        """
+        Estimate a per-pixel noise standard deviation (overestimate).
```

Comment:
What do you mean by overestimate?

Reply:
Sorry about that - for some reason I'd thought you told me that once, but I was just looking at the paper (yours and the IEEE) and you're right that it's not justified. Thanks for catching it.